src/main/java/Main.java

```diff
@@ -0,0 +1,25 @@
+import java.io.IOException;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/** A GUI for Duke using FXML. */
+public class Main extends Application {
```

Comment:
May want to group and package all your utilities files tgt (Parser, GUI, Launcher). Better code structure and understanding